tests/gold_tests/continuations/double.test.py

```diff
@@ -50,15 +48,18 @@
     'map http://127.0.0.1:{0} http://127.0.0.1:{1}'.format(ts.Variables.port, server.Variables.Port)
 )
 
-numberOfRequests = randint(1000, 1500)
+cmdstr = 'curl -k -vs http://127.0.0.1:{0}'.format(ts.Variables.port)
+numberOfRequests = 3
 
-# Make a *ton* of calls to the proxy!
+# Make calls to the proxy!
 tr = Test.AddTestRun()
-tr.Processes.Default.Command = 'ab -n {0} -c 10 http://127.0.0.1:{1}/;sleep 5'.format(numberOfRequests, ts.Variables.port)
+tr.Processes.Default.Command = 'sleep 10' # ensure stats updated
 tr.Processes.Default.ReturnCode = 0
-# time delay as proxy.config.http.wait_for_cache could be broken
 tr.Processes.Default.StartBefore(server, ready=When.PortOpen(server.Variables.Port))
 tr.Processes.Default.StartBefore(ts, ready=When.PortOpen(ts.Variables.port))
+tr.Processes.Default.StartAfter( 
+    *Test.ChainCurlCommands(*(numberOfRequests, cmdstr)) 
```

Comment:
```Python
*Test.ChainCurlCommands(numberOfRequests, cmdstr)
```
We are calling the method with two parameters and exploding the result, right?

Reply:
Yea I shouldn't be doing this. Fixed.